src/TouristInfoCards.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Card from "./Card.js";
+function TouristInfoCards(props) {
+  return (
+    <div className="cardsBlock">
+      {props.cities.map((city, index) => (
+        <Card city={city} key={index} />
+      ))}
+    </div>
+  );
+}
+export default TouristInfoCards;
```

Comment:
Nice!

Reply:
Where are you using this component? I can't find.

Reply:
that should be a container for all of the cards in App.js let me check if i can find it

Reply:
it's on line 39 in App.js ```<TouristInfoCards cities={cities} />```